Kickstarter-iOS/AppDelegate.swift

```diff
@@ -40,7 +41,7 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
     AppEnvironment.replaceCurrentEnvironment(facebookAppDelegate: FBSDKApplicationDelegate.sharedInstance())
 
     #if DEBUG
-      if Secrets.isOSS {
+      if KsApi.Secrets.isOSS {
```

Comment:
so both `LiveStream` and `KsApi` frameworks have access to the secrets, and so we need to disambiguate that. luckily both frameworks have exactly the same secrets...